### PR TITLE
[14.0][FW] shopfloor_workstation: multiple ports from 13.0

### PR DESCRIPTION
**Pull request by sebalix**
Port of the following PRs from 13.0 to 14.0:
- #173
- #169

### CODE REVIEW

File: sale_stock_available_to_promise_release/__manifest__.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "version": "13.0.1.3.0",
     "summary": "Integration between Sales and Available to Promise Release",
     "author": "Camptocamp,Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/wms",
```
Comment (sebalix) [inline sale_stock_available_to_promise_release/__manifest__.py]: Note to myself: to remove

Comment (sebalix) [reply sale_stock_available_to_promise_release/__manifest__.py]: done

**Review by mmequignon — APPROVED** [review]

**Comment by simahawk** [comment]
Probably the commit [Disable ESLint for all shopfloor modules](https://github.com/OCA/wms/pull/271/commits/4dd751587161527cee7dbd89d0982d8c1f0df5dc) is not wanted in v14 

@hparfr could you review here?

**Comment by sebalix** [comment]
@simahawk I skip it at first and Travis was not happy, so I finally included it here

**Comment by simahawk** [comment]
travis was not happy  because it implies to fix linting. Let's wait for @hparfr feedback because he worked on this subject AFAIS.
Anyway, what about shopfloor_workstation_mobile? Will it come in a new PR?

**Comment by sebalix** [comment]
@simahawk yes, but I could include it here if you prefer

**Comment by hparfr** [comment]
I already disabled (err -> warn) some stupid rules. If there some more, we can disable them as well.

Personally, I dislike most of the rules, in particular the cosmetic ones, like upper case in comments :rage:

**Comment by simahawk** [comment]
> @simahawk yes, but I could include it here if you prefer

no that's ok.



> I already disabled (err -> warn) some stupid rules. If there some more, we can disable them as well.
> 
> Personally, I dislike most of the rules, in particular the cosmetic ones, like upper case in comments rage

@hparfr ok, then, let's go like this. Would be nice to have some linting tho somehow in the future ;)

**Review by simahawk — APPROVED** [review]

**Comment by hparfr** [comment]
I mean, is better to disable problematic rules than exclude everything.

**Comment by simahawk** [comment]
ops

**Comment by sebalix** [comment]
@hparfr oh sorry, I merge too fast then, so let's fix the JS lint in another PR

**Comment by simahawk** [comment]
@sebalix no it's ok. Go ahead. I blocked my merge build... we did at the same time :smile:

**Comment by sebalix** [comment]
Work to migrate `shopfloor_workstation_mobile` is done here:
- https://github.com/OCA/wms/pull/273

**Pull request merged**